.github/workflows/ci-conan.yml

```diff
@@ -72,16 +72,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2016]
+        os: [windows-2019]
         build_type: [Debug, Release]
-        compiler_version: [15]
```

Comment:
Not sure if it is really necessary but may be we explicitly state version here?  `compiler_version: [16]`

Reply:
There is no reason to. VS2019 == 16. https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md